main.cpp

```diff
@@ -31,7 +31,7 @@
 static const int EOS_TOKEN_ID = 2;
 
 // determine number of model parts based on the dimension
-static const std::map<int, int> LLAMA_N_PARTS = {
+static const std::unordered_map<int, int> LLAMA_N_PARTS = {
```

Comment:
No. this map is just too small to have any benefit.

Reply:
actually unsure about this one, or if i am must nitpicking :)

Reply:
I agree this map is too small, but I am not sure too, looking the code it seems to be used only one time during the model load, outside the code hotspots....

Reply:
Sadly since the others maps are changed, if I rollback this map for std::map, it will be necessary to add the include <map>

Reply:
Actually, since it has no runtime performance impact, If this removes the last reference of `std::map` from the file, we could remove the `#include <map>` and get a small compile time speedup. and it looks like it is.

Reply:
> Please remove the `#include`

Done. I commited the code without the `#include <map>`